framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -2058,6 +2059,43 @@ public AnnotatedPrimitiveType getUnboxedType(AnnotatedDeclaredType type)
     }
 
     /**
+     * Returns AnnotatedDeclaredType with underlying type String and annotations copied from type.
```

Comment:
The documentation doesn't really make clear what the use of this method is.
Should it be used when something is automatically converted to String? At the moment there is only a call in an error method.